app/views/account/devices/index.html.haml

```diff
@@ -0,0 +1,23 @@
+.col-xl-6.col-12
+  .box-content
+    %h4.box-title Devices
+    / /.box-title
+    %table.table.table-striped.margin-bottom-10
+      %thead
+        %tr
+          %th{:style => "width:40%;"} Name
+          %th Type
+          %th Options          
+          %th Options
```

Comment:
what's w copy-paste?
why 2 options?

Reply:
But we have 2 options there.
Or should i just rename them?

![image](https://user-images.githubusercontent.com/48061439/59368118-305ae080-8d46-11e9-82c7-2f7066b5cf33.png)

Reply:
Imagine we have 10 options? So 10 same columns? I want to see 2 columns as now, but title merged into 1